lib/rubocop/cop/style/class_and_module_children.rb

```diff
@@ -125,14 +125,19 @@ def remove_end(corrector, body)
           corrector.remove(range)
         end
 
+        # rubocop:disable Metrics/AbcSize
         def unindent(corrector, node)
           return if node.body.children.last.nil?
 
-          column_delta = configured_indentation_width - leading_spaces(node.body.children.last).size
+          last_children_leading_spaces = leading_spaces(node.body.children.last)
```

Comment:
Since it is a single node, wouldn't `last_child` be more appropriate than `last_children`?

```suggestion
          last_child_leading_spaces = leading_spaces(node.body.children.last)
```